docs/syscall.md

```diff
@@ -0,0 +1,17 @@
+# System Calls
+
+## SDL-Oriented System Calls
```

Comment:
Replace "SDL-Oriented System Calls" with "Experimental Display and Event System Calls" and mention SDL at the beginning of this paragraph.